_build/html/_sources/capstone-report.md

```diff
@@ -75,7 +80,7 @@ Final image transformations included random vertical and horizontal flipping and
 (model-dev)=
 ### 4.2 Model Development 
 
-The augmented data is then used to train a CNN model using transfer learning. There are many popular architectures for transfer learning models that have proven successful as discussed in our literature review. As such, we investigated the following structures: VGG16, ResNet, DenseNet and Inception. Each of the models were incorporated with our small dataset, trained in separate experiments utilizing techniques to optimize the parameters of the model to maximize its ability to learn. To compare the performance of the different architectures, the team considered the accuracy and recall scores of the models when tested on our test set. 
+The augmented data is then used to train a CNN model using transfer learning, a technique utlizing pre-trianed models which allows for training with comparatively little data. Based on our literature review, the transfer learning architectures we chose to investigate were the following: VGG16, ResNet50, DenseNet169 and InceptionV3. Each of the models were incorporated with our small dataset, trained in separate experiments utilizing techniques to optimize the parameters of the model to maximize its ability to learn. To compare the performance of the different architectures, the team considered the accuracy and recall scores of the models when tested on our test set. When comparing the relative scores by looking at {numref}`acc_chart`, DenseNet outperforms the other architectures.
```

Comment:
spelling mistake: 'utilizing', 'pre-trained'

Suggestion: "a technique utilizing pre-trained models on thousands of images and then training them on similar but smaller datasets" (may not be the best but I wanted to mention the thousands of images part somehow)

Suggestion: "When comparing the relative scores of accuracy shown in {numref}`acc_chart` and recall shown in {numref}`recall_chart`, DenseNet outperforms the other architectures. 

Do we want to mention other literature that also had success with DenseNet?